Packages/com.unity.inputsystem/InputSystem/Editor/Debugger/InputDeviceDebuggerWindow.cs

```diff
@@ -143,6 +146,18 @@ internal void OnGUI()
                 EditorGUILayout.LabelField("Flags", m_DeviceFlagsString);
             if (m_Device is Keyboard)
                 EditorGUILayout.LabelField("Keyboard Layout", ((Keyboard)m_Device).keyboardLayout);
+            const string sampleFrequencyTooltip = "Displays the current event or sample frequency of this device in Hertz (Hz) averaged over measurement period of 1 second. " +
+                "The target frequency is device and backend dependent and may not be supported by all devices nor backends. " +
+                "The Polling Frequency indicates system polling target frequency.";
+            if (!string.IsNullOrEmpty(m_DeviceFrequencyString))
+                EditorGUILayout.LabelField(new GUIContent("Sample Frequency", sampleFrequencyTooltip), new GUIContent(m_DeviceFrequencyString), EditorStyles.label);
+            const string inputSystemLatencyTooltip =
+                "Displays the average/minimum/maximum observed input processing latency " +
+                "(Excluding OS, driver, firmware or transport latency) for data reported for this device. " +
+                "Note that additional experienced latency (a.k.a. input lag) will be introduced if any deferred " +
+                "processing and output delay (rendering, display refresh latency etc).";
+            if (!string.IsNullOrEmpty(m_DeviceLatencyString))
+                EditorGUILayout.LabelField(new GUIContent("Input Latency", inputSystemLatencyTooltip), new GUIContent(m_DeviceLatencyString), EditorStyles.label);
```

Comment:
I consider renaming this from "Input Latency" to "Processing Delay" to more accurately specify its the latency between first seeing the sample/event in native to actually receiving it. Thoughts?

Reply:
Maybe "Processing Delay" will do then since we are already inside "Input Debugger"

Reply:
Will mark this resolved now